cfgov/jinja2/v1/the-bureau/bureau-structure/_vars-division-data.html

```diff
@@ -0,0 +1,208 @@
+{% set data = [{
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/dca185a0d8c0205efa146b0695e88fa92ea3df97